scripts/convert_lit_checkpoint.py

```diff
@@ -124,7 +124,7 @@ def convert_lit_checkpoint(
     # checkpoint_name cannot be hardcoded because there exists different outputs such as
     # ("lit_model_finetuned.pth", "lit_model_lora_finetuned.pth", "lit_model_adapter_finetuned.pth"")
     pth_file = checkpoint_dir / checkpoint_name
-    bin_file = "".join([checkpoint_name, ".bin"])
+    bin_file = checkpoint_name.replace("pth", "bin")
```

Comment:
I suggest to simplify even more. You can 1)
```suggestion
    bin_file = pth_file.with_suffix(".bin")
```
and 2) in the save call below replace `checkpoint_dir / bin_file` with just `bin_file`

Let me know if that works.

Reply:
This works. Implemented in https://github.com/Lightning-AI/lit-gpt/pull/326/commits/5699dcd13cdf782a577beda9459fa457b320b144